core/src/module/usModuleContext.cpp

```diff
@@ -175,8 +177,14 @@ std::string ModuleContext::GetDataFile(const std::string &filename) const
 #else
     static const char separator = '/';
 #endif
-
-  std::string baseStoragePath = ModuleSettings::GetStoragePath();
+  
+  std::string baseStoragePath;
+  std::map<std::string, std::string>::iterator prop = d->module->coreCtx->frameworkProperties.find("org.osgi.framework.storage");
```

Comment:
This will be fixed as part of [this issue](https://github.com/jeffdiclemente/CppMicroServices/issues/5)